src/main/java/net/quedex/api/user/UserMessageReceiver.java

```diff
@@ -167,6 +173,18 @@ private void onOrderCancelFailed(OrderCancelFailed orderCancelFailed) {
         }
     }
 
+    private void onAllOrdersCanceled() {
+        if (orderListener != null) {
```

Comment:
Please follow the pattern from other methods. There is a race condition between checking `null` and calling the listener.

Reply:
OK